test/notebooks/testExplorer/TestExplorer.ts

```diff
@@ -0,0 +1,138 @@
+import { MessageTypes } from "../../../src/Contracts/ExplorerContracts";
+import "../../../less/hostedexplorer.less";
+import { TestExplorerParams } from "./TestExplorerParams";
+import { ClientSecretCredential } from "@azure/identity";
+import { DatabaseAccountsGetResponse } from "@azure/arm-cosmosdb/esm/models";
+import { CosmosDBManagementClient } from "@azure/arm-cosmosdb";
+import * as msRest from "@azure/ms-rest-js";
+import * as ViewModels from "../../../src/Contracts/ViewModels";
+
+class CustomSigner implements msRest.ServiceClientCredentials {
+  private token: string;
+  constructor(token: string) {
+    this.token = token;
+  }
+
+  async signRequest(webResource: msRest.WebResourceLike): Promise<msRest.WebResourceLike> {
+    webResource.headers.set("authorization", `bearer ${this.token}`);
+    return webResource;
+  }
+}
+
+const handleMessage = (event: MessageEvent): void => {
+  if (event.data.type === MessageTypes.InitTestExplorer) {
+    sendMessageToExplorerFrame(event.data);
+  }
+};
+
+const AADLogin = async (
+  notebooksTestRunnerApplicationId: string,
+  notebooksTestRunnerClientId: string,
+  notebooksTestRunnerClientSecret: string
+): Promise<string> => {
+  const credentials = new ClientSecretCredential(
+    notebooksTestRunnerApplicationId,
+    notebooksTestRunnerClientId,
+    notebooksTestRunnerClientSecret
+  );
+  const token = await credentials.getToken("https://management.core.windows.net/.default");
+  return token.token;
+};
+
+const getDatabaseAccount = async (
+  token: string,
+  notebooksAccountSubscriptonId: string,
+  notebooksAccountResourceGroup: string,
+  notebooksAccountName: string
+): Promise<DatabaseAccountsGetResponse> => {
+  const client = new CosmosDBManagementClient(new CustomSigner(token), notebooksAccountSubscriptonId);
+  return await client.databaseAccounts.get(notebooksAccountResourceGroup, notebooksAccountName);
```

Comment:
```suggestion
  return client.databaseAccounts.get(notebooksAccountResourceGroup, notebooksAccountName);
```
No await needed. I should probably add a lint rule for this.